tv_apps/fling-player/js/casting_message.js

```diff
@@ -0,0 +1,185 @@
+(function(exports) {
+  'use strict';
+
+  /**
+   * This object parses and process the message casted
+   * between receiver and controller.
+   *
+   * The message casted is in JSON and shall carry properties below:
+   * - seq: Number. The sequence of message sent. Increased by 1 when sending
+   *        one new message. The message sequences from controller and receiver
+   *        shall be independent to each other.
+   * - type: String. Indicate the message type.
+   * - other extra properties depending on the type
+   */
```

Comment:
The comment is very good
I would suggest using jsdocs format so that `make docs` generates corresponding documents.